.github/workflows/code-health.yml

```diff
@@ -70,6 +70,14 @@ jobs:
           go-version-file: 'go.mod'
       - name: website lint
         run: make tools && make website-lint
+  dependency-review:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'pull_request' }}
```

Comment:
```suggestion
   
```

I would remove the check for pr and will do always, e.g. why not do it if workflow_dispatch?